e2e/e2e-windows-scenario.sh

```diff
@@ -8,15 +8,51 @@ choose() {
     echo ${1:RANDOM%${#1}:1} $RANDOM;
 }
 
+collect-logs() {
+    local retval
+    retval=0
+    mkdir -p $SCENARIO_NAME-logs
+    VMSS_INSTANCE_ID="$(az vmss list-instances --name $DEPLOYMENT_VMSS_NAME -g $MC_RESOURCE_GROUP_NAME | jq -r '.[0].instanceId')"
+    set +x
+    expiryTime=$(date --date="2 day" +%Y-%m-%d)
+    token=$(az storage container generate-sas --account-name $STORAGE_ACCOUNT_NAME --account-key $MAPPED_ACCOUNT_KEY --permissions 'rwacdl' --expiry $expiryTime --name $STORAGE_CONTAINER_NAME --https-only)
+    # Use .ps1 file to run scripts since single quotes of parameters for --scripts would fail in check-shell
+    az vmss run-command invoke --command-id RunPowerShellScript \
+        --resource-group $MC_RESOURCE_GROUP_NAME \
+        --name $DEPLOYMENT_VMSS_NAME \
+        --instance-id $VMSS_INSTANCE_ID \
+        --scripts @upload-cse-logs.ps1 \
+        --parameters arg1=$DEPLOYMENT_VMSS_NAME arg2=$token
+    if [ "$retval" != "0" ]; then
+        echo "failed in uploading cse logs"
+    fi
+    wget https://aka.ms/downloadazcopy-v10-linux
+    tar -xvf downloadazcopy-v10-linux
+    tokenWithoutQuote=${token//\"}
+    # use array to pass shellcheck
+    array=(azcopy_*)
+    ${array[0]}/azcopy copy "https://abe2ecselog.blob.core.windows.net/cselogs/${DEPLOYMENT_VMSS_NAME}-cse.log?${tokenWithoutQuote}" $SCENARIO_NAME-logs/CustomDataSetupScript.log
```

Comment:
do you plan to replace `abe2ecselog` with `$STORAGE_ACCOUNT_NAME` in all URLs?

Reply:
I think it's better to replace it in all URLs.  Besides, I modify the container name together.